### PR TITLE
Build docs before install to debian

### PR DESCRIPTION
**Pull request by magni-mar**

### CODE REVIEW

**Comment by magni-mar** [comment]
image

File: docs/CMakeLists.txt [hunk 1]
```diff
@@ -104,6 +104,10 @@ set_target_properties(ReadTheDocs PROPERTIES
 
 # Add an install target to install the docs
 include(GNUInstallDirs)
+
+# ReadTheDocs needs to be executed before tfc-docs is installed because Doxygen needs to be executed in order for documentation to be generated
+add_custom_target(InstallDocumentation ALL DEPENDS ReadTheDocs)
+
```
Comment (omarhogni) [inline docs/CMakeLists.txt:110]: This only counteracts line 102 and makes the documentation build to often.

Comment (magni-mar) [reply docs/CMakeLists.txt:110]: this is an alternative fix: https://github.com/Skaginn3x/cmake_modules/pull/16

**Comment by jbbjarnason** [comment]
This problem is fixed here, https://github.com/Skaginn3x/cmake_modules/pull/15#discussion_r1512845412

**Pull request closed without merging**